ci/templates/projects.yaml

```diff
@@ -7,6 +7,7 @@
     github-check:
       jobs:
         - noop
+        - cifmw-base-tcib
```

Comment:
care to keep the order? It should go on line 12 actually (just before `cifmw-dev-prepare`)

Reply:
done